scripts/backend_test_shards.json

```diff
@@ -246,6 +246,8 @@
         "core.storage.translation.gae_models_test",
         "core.domain.translation_domain_test",
         "core.domain.translation_validators_test",
+        "core.platform.bulk_email.mailchimp_bulk_email_services_test",
+        "core.platform.bulk_email.dev_mode_bulk_email_services_test",
```

Comment:
Can these two tests be moved to Shard 2 or 8? Shard 11 currently takes quite a while to run.

Reply:
Done